### PR TITLE
Add queryString to users query

### PR DESCRIPTION
**Pull request by harryadel**
Added search functionality to users query
https://github.com/unchainedshop/unchained/issues/226

### CODE REVIEW

File: packages/api/resolvers/queries/users.js [hunk 1]
```diff
@@ -1,19 +1,38 @@
...
+  { limit, offset, includeGuests, queryString },
   { userId },
 ) {
   log(
     `query users ${limit} ${offset} ${includeGuests ? 'includeGuests' : ''}`,
     { userId },
   );
 
+  if (queryString) {
+    return Users.find(
```
Comment (pozylon) [inline packages/api/resolvers/queries/users.js:26]: Shouldn't we allow combining includeGuests with the queryString on the backend?

File: packages/api/resolvers/queries/users.js [hunk 2]
```diff
@@ -1,19 +1,38 @@
 import { log } from 'meteor/unchained:core-logger';
 import { Users } from 'meteor/unchained:core-users';
 
+Users.rawCollection().createIndex({
```
Comment (pozylon) [inline packages/api/resolvers/queries/users.js:4]: Please move this to the initializing function which also starts the migrations (see upcoming changes in https://github.com/unchainedshop/unchained/pull/229 for reference)

File: tests/auth-admin.test.js [hunk 3]
```diff
@@ -19,9 +19,15 @@ describe('Auth for admin users', () => {
   describe('Query.users', () => {
     beforeAll(async () => {
       const Users = db.collection('users');
+
+      Users.createIndex({
```
Comment (pozylon) [inline tests/auth-admin.test.js:23]: Tests should not create indexes, I checked and I've seen that there is also a createIndex in the products seeds but I don't get why, maybe the important thing is awaiting it?

**Review by pozylon — CHANGES_REQUESTED** [review]

File: tests/auth-admin.test.js (hunk 3 above)
Comment (harryadel) [reply tests/auth-admin.test.js:23]: Because there're tests that utilize the search functionality and they don't seem to work unless we set the indexes.

**Comment by pozylon** [comment]
LGTM

**Pull request merged**